src/asn_prim.ml

```diff
@@ -84,60 +84,51 @@ module Null : Prim with type t = unit = struct
   let random () = ()
 end
 
-module Integer : Prim with type t = Z.t = struct
+module Integer : Prim_s with type t = Cstruct.t = struct
```

Comment:
Note to myself: we change the (exposed) interface to also include `val concat : t list -> t`, `val length : t -> int` and a different `val random : ?size:int -> unit -> t` instead of `val random : unit -> t`.